.drone.star

```diff
@@ -25,7 +25,7 @@ def main(ctx):
     'repo': ctx.repo.name,
     'squishversion': {
         'latest': '6.7-20220106-1008-qt515x-linux64',
-        'fedora': '6.7-20220106-1008-qt515x-linux64',
+        'fedora': '7.1.1-qt64x-linux64',
         'qt512': '6.7-20210421-1504-qt512x-linux64',
```

Comment:
should we have an other tag `qt64` or is that not needed?

Reply:
With this, these tags will  be created/updated
```bash
owncloudci/squish:fedora
owncloudci/squish:fedora-7.1.1-qt64x-linux64
owncloudci/squish:fedora-38-7.1.1-qt64x-linux64
```